README.md

```diff
@@ -157,29 +206,41 @@ docker run \
     docker.greenkeytech.com/svtserver
 ```
 
-Once complete, you should have a JSON file for each audio file you generated. This JSON file contains the **word confusion lattice** that Discovery searches for your target phrases.
+Once complete, you should have a JSON file for each audio file you generated (e.g. `test.json` for `test.wav`). This JSON file contains the **word confusion lattice** that Discovery searches for your target phrases.
```

Comment:
**a** word confusion lattice but only if you want to add a line explaining what it is - and by explain, more like what is looks like than precisely what it does

Reply:
'This JSON file' to 'Each JSON file'